tests/misc/test_file_server.py

```diff
@@ -18,8 +18,8 @@ def _header_equal(header, name, value):
 def test_fileserver_redirect_https(host):
     path = "/path/to/dir/file.txt"
     ip = wrap_ip(host.hostname_or_ip)
-    res = commands.local_cmd(["curl", "-s", "-i", "http://" + ip + path])
-    lines = res.stdout.splitlines()
+    output = commands.local_cmd(["curl", "-s", "-i", "http://" + ip + path])
+    lines = output.stdout.strip().splitlines()
```

Comment:
this `output` name looks like a leftover from change moved to separate commit.
Introduction of `strip()` would be a separate bugfix?